internal/test/daemon/daemon.go

```diff
@@ -76,6 +76,7 @@ type Daemon struct {
 	SwarmPort       int // FIXME(vdemeester) should probably not be exported
 	DefaultAddrPool []string
 	SubnetSize      uint32
+	DatapathPort    uint32
```

Comment:
Can you rename this to `DataPathPort` (capital `Path`) to be consistent with the names elsewhere?